kolibri/plugins/learn/assets/src/views/ContentPage.vue

```diff
@@ -396,6 +429,15 @@
         message: 'Share',
         context: 'Option to share a specific file from a learning resource.',
       },
+      estimatedTime: {
+        message: '{ estimatedTime } minutes',
```

Comment:
"minutes" would need a singular form (edge case)

```suggestion
        message: 'About {estimatedTime, number, integer} {estimatedTime, plural, one {minute} other {minutes}}',
```

Reply:
Could we not use a `short` `time` type here? https://formatjs.io/docs/core-concepts/icu-syntax#time-type

Reply:
Heck yah @rtibbles - I'm going to write some tests for this and will make use of that.

@jtamiace - thoughts on Devon's suggestion about hedging?

The tooltip kind of does that hedging itself by calling the time the "suggested time to complete". 

Maybe we could consider just putting the label as the string with the time instead of using a tooltip. It isn't obvious that there is a tooltip there (no `(i)` icon to initiate it). There is plenty of space there to have a full string rather than an icon-labled string.